ManatiCR/manatime
# 27: Project new task.

**Steps to test**
- Create a new project node.
- Verify that the button 'add task' works successfully, in the project description.
- Verify that project field was pre-filled.
- Finalize the form and click save.

**DoD**
- Go to a project.
- It should have a custom ctools content type to add a new task (it should link to task node creation form with project field pre-filled).

[Issue link](https://github.com/ManatiCR/manatime/issues/27)
